app/css/general.css

```diff
@@ -83,16 +84,18 @@ html,body {
 	border-right: 4px solid #F5F5F5;
 }
 .sidebar .button {
+	opacity: .7;
 	display:flex;
 	font-size: 16px;
 	align-items:center;
 	position: relative;
 	height: 60px;
 
+	transition: all .25s;
+	-webkit-animation: fadein 2s;
```

Comment:
I would have to see it myself to be sure, but I feel like a 2sec fade in would become annoying after starting the app a few times.